conf/deployment/aws/ipi_3az_rhcos_3m_0w.yaml

```diff
@@ -0,0 +1,18 @@
+---
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+ENV_DATA:
+  platform: 'aws'
+  deployment_type: 'ipi'
+  region: 'us-east-2'
+  worker_availability_zones:
+  master_availability_zones:
+    - 'us-east-2a'
+    - 'us-east-2b'
+    - 'us-east-2c'
+  worker_replicas: 0
+  master_replicas: 3
+  master_instance_type: 'm5.4xlarge'
+REPORTING:
+  polarion:
+    deployment_id: '???'
```

Comment:
if there is no polarion id, can we remove reporting part

Reply:
added the conf file mainly for experimenting with such deployment type.
I will either add the Polarion ID or close the PR.
Adding DNM label for now